### PR TITLE
Move event type

### PR DESCRIPTION
**Pull request by yiqinguo**
Change SandboxChanged to a constant and move to the event package below.
**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/events/event.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ const (
 	FailedNodeAllocatableEnforcement     = "FailedNodeAllocatableEnforcement"
 	SuccessfulNodeAllocatableEnforcement = "NodeAllocatableEnforced"
 	UnsupportedMountOption               = "UnsupportedMountOption"
+	SandBoxChanged                       = "SandboxChanged"
```
Comment (mtaufen) [inline pkg/kubelet/events/event.go]: Please make the name of the constant match the event type string for consistency.

Comment (yiqinguo) [inline pkg/kubelet/events/event.go]: Sorry,I do not understand where it is inconsistent.

Comment (mtaufen) [inline pkg/kubelet/events/event.go]: Capitalization

Comment (mtaufen) [inline pkg/kubelet/events/event.go]: s/B/b

Comment (yiqinguo) [inline pkg/kubelet/events/event.go]: Thank you, I changed to lower case, consistent with the constant value.

**Comment by mtaufen** [comment]
/assign @yujuhong 
for approval

**Pull request merged**